src/main/java/Main.java

```diff
@@ -15,6 +21,18 @@
      */
     public static void main(String[] args) {
         DatabaseService.getDatabaseService().loadFromCSVsIfNecessary();
+        HashMap<String, ImageView> imageCache = new HashMap<>();
+        try {
+            imageCache.put("3", new ImageView(new Image(ResourceLoader.thirdFloor.openStream())));
+            imageCache.put("2", new ImageView(new Image(ResourceLoader.secondFloor.openStream())));
+            imageCache.put("1", new ImageView(new Image(ResourceLoader.firstFloor.openStream())));
+            imageCache.put("L1", new ImageView(new Image(ResourceLoader.firstLowerFloor.openStream())));
+            imageCache.put("L2", new ImageView(new Image(ResourceLoader.secondLowerFloor.openStream())));
+            imageCache.put("G", new ImageView(new Image(ResourceLoader.groundFloor.openStream())));
+        } catch(IOException e) {
+            e.printStackTrace();
+        }
+        ApplicationState.getApplicationState().setImageCache(imageCache);
```

Comment:
Are we sure we want to put image caching back in?

Reply:
He's doing it in the ApplicationState, which _should_ be better from a memory standpoint. I think the memory issues were related to the event bus issues that Maya found

Reply:
Oh I didn't even notice that. I was talking to Maya and she said she will be ready to push her fix for that later tonight